core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -179,6 +179,19 @@ package object config {
   private[spark] val EVENT_LOG_CALLSITE_LONG_FORM =
     ConfigBuilder("spark.eventLog.longForm.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_ENABLE_ROLLING =
+    ConfigBuilder("spark.eventLog.logRolling.enabled")
+      .doc("Whether rolling over event log files is enabled.  If set to true, it cuts down " +
+        "each event log file to the configured size.")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val EVENT_LOG_ROLLED_EVENT_LOG_MAX_FILE_SIZE =
```

Comment:
`EVENT_LOG` seems to be repeated twice. Maybe,  `EVENT_LOG_ROLLED_EVENT_LOG_MAX_FILE_SIZE` -> `EVENT_LOG_ROLLED_LOG_MAX_FILE_SIZE`?

Reply:
I'll apply former first, as we may need to hear the another voice for changing config name.